source/casts/14083.cast

```diff
@@ -0,0 +1,333 @@
+{"width":117,"version":2,"title":"","timestamp":1416446555,"height":13,"env":{"TERM":"xterm-256color","SHELL":"/usr/local/bin/zsh"},"duration":6.185840129852295}
+[0.15408,"o","\u001B[1m\u001B[7m%\u001B[27m\u001B[1m\u001B[0m                                                                                                                    \r \r\u001B]2;myron@Myron-Marstons-MacBook-Pro: ~/moz/plines\u0007\u001B]1;~/moz/plines\u0007"]
+[0.168781,"o","\r\u001B[0m\u001B[27m\u001B[24m\u001B[J\u001B[01;31m➜ \u001B[01;32m \u001B[36mplines \u001B[01;34mgit:(\u001B[31mrspec-3-upgrade\u001B[34m)\u001B[00m\u001B[01;34m \u001B[00m\u001B[K"]
+[0.38578,"o","b"]
+[0.513154,"o","\bbi"]
+[0.569142,"o","n"]
+[0.737288,"o","/"]
+[0.8013100000000001,"o","r"]
+[0.8575400000000001,"o","s"]
+[0.9456220000000001,"o","p"]
+[0.9772590000000001,"o","e"]
+[1.097066,"o","c"]
+[1.225478,"o"," "]
+[1.464861,"o","-"]
+[1.568549,"o","-"]
+[1.689873,"o","v"]
+[1.78477,"o","e"]
+[1.8726639999999999,"o","r"]
+[1.9210349999999998,"o","s"]
+[2.072959,"o","i"]
+[2.080947,"o","o"]
+[2.1377420000000003,"o","n"]
+[2.401078,"o","\r\r\n"]
+[2.405998,"o","\u001B]2;bin/rspec --version\u0007\u001B]1;bin/rspec\u0007"]
+[2.490792,"o","2.14.8\r\n"]
+[2.4966,"o","\u001B[1m\u001B[7m%\u001B[27m\u001B[1m\u001B[0m                                                                                                                    \r \r"]
+[2.496744,"o","\u001B]2;myron@Myron-Marstons-MacBook-Pro: ~/moz/plines\u0007\u001B]1;~/moz/plines\u0007"]
+[2.508271,"o","\r\u001B[0m\u001B[27m\u001B[24m\u001B[J\u001B[01;31m➜ \u001B[01;32m \u001B[36mplines \u001B[01;34mgit:(\u001B[31mrspec-3-upgrade\u001B[34m)\u001B[00m\u001B[01;34m \u001B[00m\u001B[K"]
+[3.0091780000000004,"o","b"]
+[3.1126510000000005,"o","\bbi"]
+[3.1544370000000006,"o","n"]
+[3.3287360000000006,"o","/"]
+[3.3764340000000006,"o","r"]
+[3.4561820000000005,"o","s"]
+[3.5285390000000003,"o","p"]
+[3.5767870000000004,"o","e"]
+[4.112447,"o","c"]
+[4.312729,"o","\r\r\n"]
+[4.313704,"o","\u001B]2;bin/rspec\u0007\u001B]1;bin/rspec\u0007"]
+[4.569617,"o","Run options: include {:focus=>true}\r\n"]
+[4.572399,"o","\r\nAll examples were filtered out; ignoring {:focus=>true}\r\n"]
+[4.595466,"o","\u001B[32m.\u001B[0m"]
+[4.6019,"o","\u001B[32m.\u001B[0m"]
+[4.607851999999999,"o","\u001B[32m.\u001B[0m"]
+[4.613734999999999,"o","\u001B[32m.\u001B[0m"]
+[4.618872999999999,"o","\u001B[32m.\u001B[0m"]
+[4.623240999999999,"o","\u001B[32m.\u001B[0m"]
+[4.629448999999999,"o","\u001B[32m.\u001B[0m"]
+[4.636333,"o","\u001B[32m.\u001B[0m"]
+[4.642002,"o","\u001B[32m.\u001B[0m"]
+[4.646478999999999,"o","\u001B[32m.\u001B[0m"]
+[4.654367999999999,"o","\u001B[32m.\u001B[0m"]
+[4.661460999999999,"o","\u001B[32m.\u001B[0m"]
+[4.678658999999999,"o","\u001B[32m.\u001B[0m"]
+[4.690181999999999,"o","\u001B[32m.\u001B[0m"]
+[4.705047999999999,"o","\u001B[32m.\u001B[0m"]
+[4.707037999999999,"o","\u001B[32m.\u001B[0m"]
+[4.707794999999999,"o","\u001B[32m.\u001B[0m"]
```

Comment:
This precision seems to be redundant.
Wondering if this will work is rounded to two digits after the floating point, i.e.
```
[4.70,"o", ....]
[4.70,"o", ....]
```
- won't it lose that second "o"?
Two digits should be enough precision to display it with 100Hz :D